demo/index.html

```diff
@@ -15,22 +15,18 @@
 
   <script src="../../sugar/dist/sugar.min.js"></script>
   <script src="../../sugar/dist/locales/fi.js"></script>
```

Comment:
This is breaking the localization demo because it expects the values in Finnish.

Reply:
Done.